test_bert_keras.py

```diff
@@ -0,0 +1,51 @@
+"""
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from bert_keras_layer import BertKerasModel
+from bert_tokenizer import *
+from sklearn.model_selection import train_test_split
+import time
+import argparse
+
+if __name__ == '__main__':
+
```

Comment:
Thanks could you explain what this test is supposed to accomplish in a comment section, i.e. "overfit a small sample to test model functionality" etc.